src/transformers/models/auto/tokenization_auto.py

```diff
@@ -183,6 +183,7 @@
                 ),
             ),
             ("megatron-bert", ("BertTokenizer", "BertTokenizerFast" if is_tokenizers_available() else None)),
+            ("mgp-str", ("MGPSTRTokenizer", "MGPSTRTokenizer" if is_tokenizers_available() else None)),
```

Comment:
```suggestion
            ("mgp-str", ("MGPSTRTokenizer", None)),
```
Doesn't seem like this model has a fast tokenizer